tests/ir/test_print.py

```diff
@@ -105,47 +105,47 @@ def test_format_exc():
         assert (
             format_exc(exc.value, mode="caret")
             == """File ./tests/ir/test_print.py, lines 89-92
-In fun:clone(a::*int(), b::*float())
-89     c = add(
-           ^^^^
+In fun:clone(a~29::*int(), b~29::*float())
+89     c = and_(
+           ^^^^^
 90         a,
            ^^
 91         b,
            ^^
 92     )
        ^
 
-File ./tests/ir/test_print.py, line 85
-In add:clone(x::*int(), y::*float())
-85     return x + y
-              ^^^^^
+File ./tests/test_parser.py, line 2024
+In and_:clone(x~2::*int(), y::*float())
+2024     return f"value is {x}"
+         ^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Why is it highlighting a line in `test_parser`?

Reply:
I still don't know, neither.